rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.55"
+channel = "1.56"
```

Comment:
```suggestion
channel = "1.56.1"
```

I believe that that's the current stable

Reply:
It will auto-pick the highest patch release AFAIK.